src/service/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+if (ANDROID)
+  add_library(qfield_service SHARED
+    main.cpp
+  )
+endif()
+
+#target_compile_definitions(qfield_service
+#  PRIVATE $<$<OR:$<CONFIG:Debug>,$<CONFIG:RelWithDebInfo>>:QT_QML_DEBUG>)
+
+if (ANDROID)
+    target_link_libraries(qfield_service PRIVATE
+      android
+      log
+    )
+endif()
+
+target_link_libraries(qfield_service PRIVATE
```

Comment:
what happens here if not on android?
the android check should be made one level up, no?